lib/src/containerenv.rs

```diff
@@ -20,6 +20,10 @@ pub(crate) struct ContainerExecutionInfo {
     pub(crate) rootless: Option<String>,
 }
 
+pub(crate) fn is_container(rootfs: &Dir) -> bool {
+    rootfs.exists(PATH)
```

Comment:
Hmm but we have `get_container_execution_info()` (defined just below) that we should be calling but somehow aren't fatally erroring out on right?

I'm confused since it sure looks like we *should* be calling that function and erroring out when run directly...

Reply:
yea, I can add a test case to make sure we bail out. I tested it manually and it was working. I split this into a separate function just so it can be added to the global state and eventually used to toggle selinux/kargs/etc. checks.